telebot/apihelper.py

```diff
@@ -126,9 +166,10 @@ def get_updates(token, offset=None, limit=None, timeout=None):
         payload['offset'] = offset
     if limit:
         payload['limit'] = limit
-    if timeout:
+    if timeout is not None:
```

Comment:
by the way, `timeout=0` is perfectly legal value, and telegram API says that in such case it will be 'normal, short polling', i.e. response will be given immediately.